.github/workflows/lighthouse-report.yml

```diff
@@ -1,4 +1,4 @@
-name: Lighthouse Report
+name: "CI - Lighthouse Report"
```

Comment:
This change does nothing to solve the original problem, which is that if you have never heard of 'Lighthouse' then this name is meaningless.

How about 

"CI - Lighthouse website audit"

It is a pity that workflows can't have a Description: to explain them, unlike some other things in the .yaml file.

Reply:
I know right! I wish we could have descriptions here